### ISSUE TITLE
sample "Ingest Custom API Data into Microsoft 365 with a Microsoft Graph Connector" manifest validation failed

### ISSUE THREAD

**Issue by ayachensiyuan** [pnp/graph-connectors-samples#66]
- use teams toolkit to create sample "Ingest Custom API Data into Microsoft 365 with a Microsoft Graph Connector"
- follow readme to prepare prerequisites
- local debug
![image](https://github.com/user-attachments/assets/96deba03-84a9-4a49-ae59-443d96e0629b)

ttk version: alpha / beta

**Comment by waldekmastykarz** [comment]
Thanks for reporting the issue @ayachensiyuan. Which sample (folder name) are you seeing this issue in? Do you have any more information about the validation error that could help us understand what's wrong?

**Comment by ayachensiyuan** [comment]
https://github.com/pnp/graph-connectors-samples/tree/main/samples/nodejs-typescript-food-catalog
this sample which shows in teams toolkit sample gallery
<img width="1238" alt="image" src="https://github.com/user-attachments/assets/17237a14-de0c-4f7f-9b1a-5e352920ec4c">
repro steps:
- install vsc teams toolkit beta version
- clone sample project open in vsc
- On macOS, make the devtunnel.sh script, executable, by running in command line: chmod +x ./scripts/devtunnel.sh
- Press F5, follow the sign in prompts

**Comment by waldekmastykarz** [comment]
Do you have any more information about the validation errors?

**Comment by ayachensiyuan** [comment]
> Do you have any more information about the validation errors?

nope... only got the above error message and failed to provision.

**Comment by waldekmastykarz** [comment]
Often there's more information if you scroll up in the output window, but if you don't get any more information, that's ok too. Let us investigate and we'll see if we can find out what's wrong.

**Comment by ayachensiyuan** [comment]
@waldekmastykarz This error log is not very clear.
<img width="1299" alt="image" src="https://github.com/user-attachments/assets/fc11bf88-438b-4533-bd6a-11c2f4884066">

**Comment by waldekmastykarz** [comment]
Understood. Let me have a look at it and see if I can find out more.

**Comment by ayachensiyuan** [comment]
any update for this issue? @waldekmastykarz

**Comment by waldekmastykarz** [comment]
No yet. Sorry it's taking so long. Seems like there's been a change in the feature responsible for validating solutions. I'm working on understanding what's wrong exactly and what's the best way forward.

**Comment by waldekmastykarz** [comment]
It took us way longer than intended but all seems to be working again @ayachensiyuan. Sorry for the trouble and thanks again for your patience and bringing this to our attention.
